ray dashboard command for clusters (#6959)
ray-project/ray: doc/source/autoscaling.rst

If you want to run applications on the cluster that are accessible from a web browser (e.g., Jupyter notebook), you can use the ``--port-forward`` option for ``ray exec``. The local port opened is the same as the remote port.

> Note: For Kubernetes clusters, the ``port-forward`` option cannot be used while executing a command. To port forward and run a command you need to call ``ray exec`` twice. 

Reviewer: ```suggestion
Note: For Kubernetes clusters, the ``port-forward`` option cannot be used while executing a command. To port forward and run a command you need to call ``ray exec`` twice separately.
```

Monitoring cluster status
~~~~~~~~~~~~~~~~~~~~~~~~~

> The autoscaler also comes with an online dashboard. To access the dashboard, which is located on the ray head node on port ``8265``. To access it, forward the port to your local machine. You can also call ``ray dashboard`` to do this. 

Reviewer: The wording of this is sort of weird - ray comes with a dashboard (not the autoscaler) and the dashboard isn't online.

> The autoscaler also comes with an online dashboard. To access the dashboard, which is located on the ray head node on port ``8265``. To access it, forward the port to your local machine. You can also call ``ray dashboard`` to do this. 

Reviewer: ```suggestion
The autoscaler also comes with an online dashboard. To access the dashboard, which is located on the ray head node on port ``8265``. To access it locally, you'll need to forward the port to your local machine. You can also use the built-in ``ray dashboard`` to do this automatically.
```

> The ray also comes with an online dashboard. The dashboard is accessible via HTTP on the head node (by default it listens on ``localhost:8265``). To access it locally, you'll need to forward the port to your local machine. You can also use the built-in ``ray dashboard`` to do this automatically.

Reviewer: ```suggestion
The ray also comes with an online dashboard. The dashboard is accessible via HTTP on the head node (by default it listens on ``localhost:8265``). To access it locally, you'll need to forward the port to your local machine. You can also use the built-in ``ray dashboard`` command to do this automatically.
```

forgot word
